README.md

```diff
@@ -0,0 +1,126 @@
+# coursebuilder-student-dashboard
+
+This is a Course Builder module which allows students to view and manage their courses and lessons 
+in a personalised way. It keeps your lessons organized in one place and also keep track of your progress on each lesson.
+
+The student dashboard is a custom module that was developed as a standalone system i.e 
+changes to the course builder codebase was avoided where possible. The module can be integrated 
+into the Course builder without breaking anything in the application.
+
+## Requirements
+
+You'll need a system with current versions of `bash` and `git`, as well as
+`python` 2.7.
+
+## Getting started
+
+First, clone Course Builder and change directory to the Course Builder root:
+
+```sh
+git clone https://github.com/google/coursebuilder-core.git
+cd coursebuilder-core/coursebuilder
+```
+
+Course Builder provides a management script for fetching modules, so you always
+start by grabbing Course Builder, then using it to fetch the module you want to
+work with. Let's grab the student dashboard module, called `coursebuilder-student-dashboard`.
+
+Next: create a file titled install.sh. For instance, you can place the file in your desktop and copy the script below into it.
+
+```
+  #!/bin/bash
+  set -e 
+  set -o pipefail
+  sh scripts/modules.sh \
+    --targets=coursebuilder-student-dashboard@https://github.com/andela-iakande/coursebuilder-student-dashboard.git
+
+  sleep 15 #pause shell script
+
+  cat modules/coursebuilder-student-dashboard/top-bar.html > modules/explorer/_static/components/top-bar/top-bar.html
```

Comment:
@andela-iakande Is there a work around to this so that we are not replacing static files in another module?

Reply:
@chegejames, the only thing that changes in the top bar is the href that links to the student dashboard on the explorer page.The alternative to this is to add more detailed instructions to the README file that will include the href to be inserted on the explorer page which will link to the student dashboard module since it is developers that are expected to use the module. Or what do you think?

Reply:
@andela-iakande You will also need to add this to the course view which is yet another module e.g here https://im-dot-cfafrica-mooc-dev.appspot.com/test/unit?unit=4&lesson=10

Reply:
@chegejames I get you pertaining to adding the student dashboard link to the course view as well but what mode do you suggest I should employ to add the link?